contracts/consensus-gateway/ConsensusCoGateway.sol

```diff
@@ -0,0 +1,85 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import "../consensus/CoConsensusModule.sol";
+import "../proxies/MasterCopyNonUpgradable.sol";
+import "../message-bus/MessageBus.sol";
+import "../message-bus/StateRootI.sol";
+import "../consensus-gateway/ConsensusGatewayBase.sol";
+import "../consensus-gateway/ERC20GatewayBase.sol";
+import "../consensus/CoConsensusI.sol";
+
+contract ConsensusCoGateway is MasterCopyNonUpgradable, MessageBus, ConsensusGatewayBase, ERC20GatewayBase, CoConsensusModule {
+
+    /* Constants */
+
+    uint8 constant public OUTBOX_OFFSET = 1;
+
+    uint8 constant public INBOX_OFFSET = 4;
```

Comment:
We should explicitly define it as `uint8 constant public INBOX_OFFSET = uint8(4);`

Reply:
Done